public/css/alert.css

```diff
@@ -12,18 +12,17 @@
 	border-radius: 0px;
 	outline: 0;
 	box-shadow: 0 0px 0px;
-	padding-bottom: 0px !important;
+	position: relative;
+	height: 100%;
 }
 
 .modal-content h3 {
-	font-family: 'open sans';
 	color: #575757;
 	font-weight: 600;
 	font-size: 30px;
 }
 
 .modal-content p {
-	font-family: 'open sans';
 	color: #797979;
 	font-size: 16px;
     font-weight: 300;
```

Comment:
might as well fix this indentation

Reply:
i doesn't look broken on my editor... i re-did the indent there so hopefully it fixes it :/